src/main.cpp

```diff
@@ -200,4 +201,9 @@ void loop()
         send2Floats(battery, batteryLevel, PACKET_BATTERY_LEVEL);
     }
 #endif
+    if(now_ms - last_rssi_sample >= 2000) {
+        last_rssi_sample = now_ms;
+        int8_t signalStrength = WiFi.RSSI();
+        sendByte(signalStrength, 0, PACKET_SIGNAL_STRENGTH);
```

Comment:
Did you miss PACKET_SIGNAL_STRENGTH definition in `udpclient.h`?

Reply:
Oh, yeah, missed this define when staged my changes. Added it.